app/controllers/application_controller.rb

```diff
@@ -102,6 +102,7 @@ def send_mixpanel_event(event_name:, data: {})
       intake_source: current_intake&.source,
       intake_referrer: current_intake&.referrer,
       intake_referrer_domain: current_intake&.referrer_domain,
+      **(current_intake.present? ? current_intake.mixpanel_data : {})
```

Comment:
Hmm, using the double splat operator is a using a new pattern for merging hashes - we could extend the existing pattern of using `Hash#merge` instead with something like this (on the next line):

```ruby
default_data.merge!(current_intake.mixpanel_data) if current_intake.present?
```


Also, something to consider would be to move the other `current_intake` fields above into the `mixpanel_data` method (the `source`, `referrer`, and `referrer_domain`)

Reply:
sorry, i'm completely missing the `mixpanel_data` method? is it in this file?